src/modules/tests/test-data/cat-c/test-data.cat-c.selector.ts

```diff
@@ -0,0 +1,93 @@
+import { Competencies, LegalRequirements } from '../test-data.constants';
+import { get } from 'lodash';
+import { CompetencyOutcome } from '../../../../shared/models/competency-outcome';
+import { CatCUniqueTypes } from '@dvsa/mes-test-schema/categories/C';
+import { QuestionProvider } from '../../../../providers/question/question';
+import { VehicleChecksQuestion } from '../../../../providers/question/vehicle-checks-question.model';
+import { NUMBER_OF_SHOW_ME_QUESTIONS }
+  from '../../../../shared/constants/show-me-questions/show-me-questions.cat-be.constants';
+import { NUMBER_OF_TELL_ME_QUESTIONS }
+  from '../../../../shared/constants/tell-me-questions/tell-me-questions.cat-be.constants';
+import { TestCategory } from '@dvsa/mes-test-schema/category-definitions/common/test-category';
+
+export const getDrivingFaultCount = (
+  data: CatCUniqueTypes.TestData, competency: Competencies) => data.drivingFaults[competency];
+
+export const getManoeuvres = (data: CatCUniqueTypes.TestData): CatCUniqueTypes.Manoeuvres => data.manoeuvres;
+
+// TODO - We should pass a Manoeuvre object here instead of TestData
+export const hasManoeuvreBeenCompletedCatBE = (data: CatCUniqueTypes.TestData) => {
+  return (
+    get(data.manoeuvres, 'reverseLeft.selected')
+  );
+};
+
+export const hasLegalRequirementBeenCompleted = (
+  data: CatCUniqueTypes.TestRequirements, legalRequirement: LegalRequirements) => {
+  return data[legalRequirement];
+};
+
+export const getVehicleChecks = (
+  state: CatCUniqueTypes.TestData): CatCUniqueTypes.VehicleChecks => state.vehicleChecks;
+
+export const getTellMeQuestion = (state: CatCUniqueTypes.VehicleChecks): VehicleChecksQuestion => {
+  const questionProvider: QuestionProvider = new QuestionProvider();
+  return questionProvider
+    .getTellMeQuestions(TestCategory.BE)
```

Comment:
passing TestCategory.BE to getTellMeQuestions (realise we may not have cat C questions set up in the questionprovider yet. (perhaps a todo)

Reply:
Will change it to C, but the getTellMeQuestions has to be implemented for Cat C still